nav_msgs/OccupancyGrid grid1
nav_msgs/OccupancyGrid grid2
---
geometry_msgs/Transform transform
float64 confidence